KeyVaultCA.Web/KeyVaultCA.Web.csproj

```diff
@@ -7,6 +7,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.20.0" />
 		<PackageReference Include="Microsoft.AspNetCore.Authentication.Certificate" Version="6.0.1" />
 		<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.1" NoWarn="NU1605" />
 		<PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="6.0.1" NoWarn="NU1605" />
```

Comment:
When trying to publish the app from VS2022, I ran into this issue https://stackoverflow.com/questions/71269358/in-visual-studio-2022-publish-to-iis-returns-a-build-error-but-publish-to-fold so I downgraded the 'Microsoft.VisualStudio.Azure.Containers.Tools.Targets' to 1.11.1 on my local machine. I had no issues with VS2019.